From 9c37d4206112d83196906feef04efafd2c934977 Mon Sep 17 00:00:00 2001
From: stm7128 <114091453+stm7128@users.noreply.github.com>
Date: Thu, 30 Jul 2026 16:11:57 +0900
Subject: [PATCH] a

---
 dist/windows/strawberry.nsi.in | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/dist/windows/strawberry.nsi.in b/dist/windows/strawberry.nsi.in
index 08777a86b..551f8eabc 100644
--- a/dist/windows/strawberry.nsi.in
+++ b/dist/windows/strawberry.nsi.in
@@ -13,6 +13,8 @@
   !define arch_x64
 !else if "@TARGET_ARCH@" == "arm64"
   !define arch_arm64
+!else if "@TARGET_ARCH@" == "i686"
+  !define arch_x86
 !else if "@TARGET_ARCH@" == ""
   !error "Missing TARGET_ARCH"
 !else
@@ -27,6 +29,10 @@
   !define arch "arm64"
 !endif
 
+!ifdef arch_x86
+  !define arch "x86"
+!endif
+
 
 !if "@CMAKE_BUILD_TYPE@" == "Release"
   !define release
-- 
2.55.0.windows.3

