--- dist/windows/strawberry.nsi.in
+++ dist/windows/strawberry.nsi.in
@@ -319,7 +319,6 @@
   File "liborc-0.4-0.dll"
   File "libpng16-16.dll"
   File "libpsl-5.dll"
-  File "libqtsparkle-qt6.dll"
   File "libsoup-3.0-0.dll"
   File "libspeex-1.dll"
   File "libsqlite3-0.dll"
@@ -417,7 +416,6 @@
   File "opus.dll"
   File "orc-0.4-0.dll"
   File "psl-5.dll"
-  File "qtsparkle-qt6.dll"
   File "soup-3.0-0.dll"
   File "sqlite3.dll"
   File "tag.dll"
@@ -843,7 +841,6 @@
   Delete "$INSTDIR\liborc-0.4-0.dll"
   Delete "$INSTDIR\libpng16-16.dll"
   Delete "$INSTDIR\libpsl-5.dll"
-  Delete "$INSTDIR\libqtsparkle-qt6.dll"
   Delete "$INSTDIR\libsoup-3.0-0.dll"
   Delete "$INSTDIR\libspeex-1.dll"
   Delete "$INSTDIR\libsqlite3-0.dll"
@@ -941,7 +938,6 @@
   Delete "$INSTDIR\opus.dll"
   Delete "$INSTDIR\orc-0.4-0.dll"
   Delete "$INSTDIR\psl-5.dll"
-  Delete "$INSTDIR\qtsparkle-qt6.dll"
   Delete "$INSTDIR\soup-3.0-0.dll"
   Delete "$INSTDIR\sqlite3.dll"
   Delete "$INSTDIR\tag.dll"
